sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -7,36 +7,95 @@
 namespace Azure.AI.FormRecognizer.Training
 {
     /// <summary>
+    /// Represents a model trained from custom forms.
     /// </summary>
     public class CustomFormModel
     {
-        internal CustomFormModel()
+        internal CustomFormModel(Model_internal model)
         {
+            ModelId = model.ModelInfo.ModelId.ToString();
+            Status = model.ModelInfo.Status;
+            CreatedOn = model.ModelInfo.CreatedDateTime;
+            LastModified = model.ModelInfo.LastUpdatedDateTime;
+            Models = ConvertToSubmodels(model);
+            TrainingDocuments = model.TrainResult.TrainingDocuments;
         }
 
         /// <summary>
+        /// Model identifier.
         /// </summary>
         public string ModelId { get; }
 
         /// <summary>
+        /// Status indicating the model's readiness for use.
         /// </summary>
         public CustomFormModelStatus Status { get; }
 
         /// <summary>
+        /// Date and time (UTC) when model training was started.
         /// </summary>
-        public DateTime CreatedOn { get; }
+        public DateTimeOffset CreatedOn { get; }
 
         /// <summary>
+        /// Date and time (UTC) when model training completed.
         /// </summary>
-        public DateTime LastUpdatedOn { get; }
+        public DateTimeOffset LastModified { get; }
```

Comment:
Is this a change we're doing across language?

Reply:
asked on Teams channel